src/streams/SubscriptionsScreen.js

```diff
@@ -60,7 +60,8 @@ export default function SubscriptionsScreen(props: Props): Node {
       .sort((a, b) => caseInsensitiveCompareFunc(a.name, b.name));
     return [
       { key: 'Pinned', data: sortedSubscriptions.filter(x => x.pin_to_top) },
-      { key: 'Unpinned', data: sortedSubscriptions.filter(x => !x.pin_to_top) },
+      { key: 'Unpinned', data: sortedSubscriptions.filter(x => !x.pin_to_top && x.in_home_view) },
+      { key: 'muted', data: sortedSubscriptions.filter(x => !x.in_home_view) },
```

Comment:
Should "muted" here not be "Muted"?

Reply:
Yeah. I have made changes. Review it. @cyphase